cmd/clusterctl/client/cluster/template_test.go

```diff
@@ -42,6 +42,21 @@ kind: Cluster
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: Machine`
 
+var rawURLTemplate = `apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+  - name: command-demo-container
+    image: debian
+    command: ["printenv"]
+    args: ["HOSTNAME", "KUBERNETES_PORT"]
+  restartPolicy: OnFailure
+`
+
```

Comment:
I have added this just for testing purpose as I don't have any raw URL in my hand that contains the cluster template I will update this if there is one